include/internal/sockets.h

```diff
@@ -72,11 +72,7 @@ struct servent *PASCAL getservbyname(const char *, const char *);
 #  else
 #   include <sys/socket.h>
 #   ifndef NO_SYS_UN_H
-#    ifdef OPENSSL_SYS_VXWORKS
```

Comment:
Will this change break older versions of Vxworks...and if so, how old?

Reply:
it shouldnt, all i did in this pullrequest is to sort out the #ifdefs for vxworks to be consistent